### PR TITLE
Addressing minor aggregator TODOs

### PR DESCRIPTION
**Pull request by vusirikala**
### Description

Removing outdated aggregator TODOs and addressing a minor cleanup aggregator TODO. 
There was one TODO in `test_change_set.rs` that asked to consolidate tests from 2 files. While addressing this change I had to indent a few test cases inside `mod tests` resulting in high line count for the PR. Otherwise, it's a very small PR.

### CODE REVIEW

File: aptos-move/aptos-aggregator/src/types.rs
Comment (igor-aptos) [inline aptos-move/aptos-aggregator/src/types.rs:10]: this TODO is to remove the "pub" from the line below, and have everyone directly import those from aptos_types

File: aptos-move/aptos-vm-types/src/tests/test_change_set.rs
Comment (igor-aptos) [inline aptos-move/aptos-vm-types/src/tests/test_change_set.rs:4]: argh github annotates your change here terribly

I trust you know what you changed, if you want review - comment on the part of the code that actually got changed

File: aptos-move/block-executor/src/view.rs
Comment (igor-aptos) [inline aptos-move/block-executor/src/view.rs:3237]: can you take these and make them tests? these are your original tests

**Review by igor-aptos — CHANGES_REQUESTED** [review]

File: aptos-move/aptos-aggregator/src/types.rs
Comment (georgemitenkov) [reply aptos-move/aptos-aggregator/src/types.rs:10]: I would also wait for delayed value impl PR to land first, I had to move some things to move third_party, so it will be easier to consolidate all imports after

Comment (vusirikala) [reply aptos-move/aptos-aggregator/src/types.rs:10]: I've made the changes Igor mentioned. Will update again after your PR lands.

**Comment by vusirikala** [comment]
> @vusirikala can you rebase?

I actually addressed the same TODO in another PR. So, closing this PR.

**Pull request closed without merging**
